build/package/github-actions/github-actions.go

```diff
@@ -124,17 +124,26 @@ func parseBranch(ref string, event *Event, allowTags bool) (string, error) {
 	}
 	results := re.FindStringSubmatch(ref)
 
-	if results == nil {
-		// The GITHUB_REF wasn't valid, so check if it's a pull request and use the pull request ref instead
-		if event != nil && event.Pull != nil {
-			return event.Pull.Head.Ref, nil
-		}
+	if results != nil {
+		return results[name_index], nil
+	}
 
-		if allowTags && event != nil && event.Release != nil {
-			return event.Release.TagName, nil
-		}
+	// The GITHUB_REF wasn't valid, so check if it's a pull request and use the pull request ref instead
+	if event != nil && event.Pull != nil {
+		return event.Pull.Head.Ref, nil
+	}
+
+	// If it's not a pull request, check if it's a Release
+	if allowTags && event != nil && event.Release != nil {
```

Comment:
does `event.Release` also cover just creating tags without an associated release?

Reply:
no, but that should be caught in the block above with the `ref` which is used for push events - as far as i can tell a tag would only come into play from a `push` and `release` - anything you think i'm missing? i'll do another test though

Reply:
Yep that makes sense!